src/components/DatePicker/index.tsx

```diff
@@ -35,7 +35,7 @@ type DatePickerProps = {
     maxDate?: Date;
 
     /** A function that is passed by FormWrapper */
-    onInputChange?: (value: Date) => void;
+    onInputChange?: (value: string) => void;
```

Comment:
```suggestion
    onInputChange?: (value: Date) => void;
```

Can we avoid this? Why do we need to change to `string`?

Reply:
@fabioh8010 this type was wrong in the first place, we never pass a `Date` object as an argument, we always pass a `string`.